tower-backend/src/main/resources/application.yml

```diff
@@ -3,7 +3,7 @@ tower:
     serverUrl: "${TOWER_SERVER_URL:`http://localhost:8000`}"
     contactEmail: "${TOWER_CONTACT_EMAIL:hello@tower.nf}"
     trustedEmails:
-      - '*@seqera.io'
+      - '*'
```

Comment:
Not sure if you would rather leave this filled? I guess I should remove the bit in the readme about updating this if we leave it like this..

Reply:
I'm changing this so that any user is whited listed by default and optional `trustedEmails` can be provided to enable this feature.

Reply:
Updated PR to solve merge conflicts ✨